src/style.css

```
@@ -0,0 +1,135 @@
+* {
+  box-sizing: border-box;
+}
+
+h1,
+h2,
+h3,
+p,
+a,
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.page {
+  font-family: "Roboto", sans-serif;
+}
+
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  font-family: "Roboto", sans-serif;
+  color: #060b35;
+}
+
+.card__image {
+  display: block;
+  width: 160px;
+  height: 134px;
+  margin: 0 auto;
+  margin-bottom: 40px;
+}
```

давай поєднаймо ці стилі
```suggestion
  margin: 0 auto 40px;
}
```

---

done